examples/nemo/scripts/kernels_trans.py

```diff
@@ -670,7 +670,7 @@ def trans(psy):
             add_kernels(sched.children)
         else:
             print("Addition of OpenACC to routine {0} disabled!".
```

Comment:
We're in the process of updating to use f-strings on a "you touch it, you change it" basis so please could you change this to use an f-string instead of format.

Reply:
Done as part of the new commit.